examples/status.rs

```diff
@@ -0,0 +1,42 @@
+use bevy::prelude::*;
+use bevy_kira_audio::{Audio, AudioPlugin, InstanceHandle};
+
+struct LoopAudioInstanceHandle {
+    instance_handle: InstanceHandle,
+}
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_plugin(AudioPlugin)
+        .add_startup_system(start_audio.system())
+        .add_system(print_status.system())
+        .add_system(process_keyboard_input.system())
+        .run();
+}
+
+fn start_audio(mut commands: Commands, asset_server: Res<AssetServer>, audio: Res<Audio>) {
+    let asset_handle = asset_server.load("sounds/loop.ogg");
+    let instance_handle = audio.play(asset_handle);
+    println!("Audio started.");
+    std::thread::sleep(std::time::Duration::from_secs(5));
```

Comment:
Why do we sleep?

Reply:
At one point I didn't think the audio was starting correctly without it, but now I remove the sleep that is not the case. I'll take it out.